pint/util.py

```diff
@@ -876,7 +880,11 @@ def getattr_maybe_raise(self, item):
     """
     # Double-underscore attributes are tricky to detect because they are
     # automatically prefixed with the class name - which may be a subclass of self
-    if item.startswith("_") or item.endswith("__"):
+    if (
+        item.startswith("_")
```

Comment:
I would add a parenthesis with the first two clauses. It will help to indicate purpose and link to the explanation in the docs. i.e I think the intention would be clear if
```python
item.endswith("__") 
or (item.startswith("_") and not item.lstrip("_")[0].isdigit())
```

Also the above introduces a subtle bug. If `item == "_"`, `IndexError` will be raised.